addons/mail/static/src/core/common/suggestion_service.js

```diff
@@ -36,10 +36,7 @@ export class SuggestionService {
         const cleanedSearchTerm = cleanTerm(term);
         switch (delimiter) {
             case "@": {
-                await Promise.all([
-                    this.fetchPartners(cleanedSearchTerm, thread, { abortSignal }),
-                    this.store.fetchStoreData("res.role", { term: cleanedSearchTerm }),
-                ]);
```

Comment:
I would keep the 2 separate parameters, so we can easily call one without the other if/when we need.
`fetchStoreData` is taking care of the grouping so that the business code doesn't have to worry about it.